packages/thrift-client-timing-filter/tsconfig.json

```diff
@@ -12,7 +12,8 @@
         "strict": true,
         "noUnusedLocals": true,
         "pretty": true,
-        "removeComments": true
+        "removeComments": true,
+        "useUnknownInCatchVariables": false
```

Comment:
Without `"useUnknownInCatchVariables": false` tsc continued to revert `err` types to become typeless despite being updated to `any`. Assuming this has to do with `"strict": true`. An alternative to this appears to be casting `err` types to `Error` before accessing them (i.e., `((e as Error).message)`).